code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -347,6 +347,10 @@
 	if(user.real_name == "[originalname][temporaryname]" || user.real_name == "A Horse With No Name") //if it's somehow changed while the mask is on it doesn't revert
 		user.real_name = originalname
 
+/obj/item/clothing/mask/horsehead/change_speech_verb()
+	if (voicechange == TRUE)
```

Comment:
Since it's a boolean, this can be shortened to just `if(voicechange)`.
```suggestion
	if(voicechange)
```
(Also spacing after the if)